controllers/testpipeline/testpipeline_controller.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions andF
 limitations under the License.
 */
 
-package pipeline
+package testpipeline
```

Comment:
Can we call this `integrationpipeline` instead?  It lines up with the name of our service, but since there are also tests in the build pipeline it draws a clear line regarding what's being tested.

Reply:
Oh, it was changed back to `pipeline` when I rebase the code. Thanks!

Reply:
Sorry. I misunderstood your comment. Updated. Thanks!